### PR TITLE
Release Notes for September 2023

### PR DESCRIPTION
**Pull request by elisekain**

### CODE REVIEW

**Review by HBehrens — COMMENTED** [review]
Generally LGTM.

Two remarks:
1. Bullet points on "General" section got lost.
2. Capitalization is off in some places (e.g. "Device Timeline" is what we generally write instead of "device timeline" in this version.

File: blog/2023-09-30.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+<!-- truncate -->
+
+## General
+
+Added: New swimlane on the device timeline to display the software versions over
+time.
+
+Added: New
+“[Using MQTT with Memfault](/docs/best-practices/mqtt-with-memfault/)” best
+practices doc
```
[the remark this answers is not in the document]
Comment (elisekain) [reply blog/2023-09-30.mdx]: Fixed! Thank you.

**Comment by HBehrens** [comment]
Thank you, @elisekain – something is still off with that list though. I attached a screenshot from this PR's preview and our last changelog for comparison.
CleanShot 2023-10-09 at 15 25 50

**Comment by elisekain** [comment]
> Thank you, @elisekain – something is still off with that list though. I attached a screenshot from this PR's preview and our last changelog for comparison. CleanShot 2023-10-09 at 15 25 50

@HBehrens I really think I've got it this time - sorry for the back and forth on something this simple.

**Review by HBehrens — APPROVED** [review]

**Comment by HBehrens** [comment]
Sorry for the nitpicking and so much back and forth – will merge now, @elisekain

**Pull request merged**